assets/scripts/lobby/chat4.js

```diff
@@ -282,9 +291,16 @@ function addToRoomChat(data) {
                 if(filteredMessage.indexOf("[") === -1){
                     //set the highlight chat if the user has been selected already
                     var highlightChatColour = "";
+                    var setHighlightColorToYellow = $(".setHighlightColorsToYellow")[0].checked;
```

Comment:
You're not making sure that both buttons are always the same value. If I check the box in tab 1, it should also be automatically checked in tab 2. If I check the box in tab 2, the box in tab 1 isn't checked, and `$(".setHighlightColorsToYellow")[0].checked;` would return false.

Check to see how I made the mute spectators button bind to each other. 

Also, the button seems to not be updating all the previous chat that has already loaded. Results in this:
![image](https://user-images.githubusercontent.com/42633156/45153815-add24d00-b218-11e8-8c09-57ec28a7c22f.png)